ExampleApps/SwiftUIExample/SwiftUIExampleTests/UIKitInteropTests.swift

```diff
@@ -0,0 +1,213 @@
+            let wrapper = try workflowLauncher.view(ViewControllerWrapper<TestInputViewController>.self)
+            let context = unsafeBitCast(FakeContext(), to: UIViewControllerRepresentableContext<ViewControllerWrapper<TestInputViewController>>.self)
+            vc = try wrapper.actualView().makeUIViewController(context: context)
+        }
+
+        wait(for: [exp], timeout: TestConstant.timeout)
+
+        vc.loadOnDevice()
+
+        XCTAssertUIViewControllerDisplayed(isInstance: vc)
+
+        let proceedCalled = expectation(description: "proceedCalled")
+        vc.proceedInWorkflowStorage = { _ in
+            proceedCalled.fulfill()
+        }
+
+        vc.proceedInWorkflow()
+
+        wait(for: [proceedCalled], timeout: TestConstant.timeout)
+    }
+
+    func testPuttingAUIKitViewFromStoryboardThatDoesNotTakeInDataInsideASwiftUIWorkflow() throws {
+        let workflowView = WorkflowLauncher(isLaunched: .constant(true))
+            .thenProceed(with: WorkflowItem(TestNoInputViewController.self))
+        var vc: TestNoInputViewController!
+
+        let exp = ViewHosting.loadView(workflowView).inspection.inspect { workflowLauncher in
+            let wrapper = try workflowLauncher.view(ViewControllerWrapper<TestNoInputViewController>.self)
+            let context = unsafeBitCast(FakeContext(), to: UIViewControllerRepresentableContext<ViewControllerWrapper<TestNoInputViewController>>.self)
+            vc = try wrapper.actualView().makeUIViewController(context: context)
+        }
+
+        wait(for: [exp], timeout: TestConstant.timeout)
+
+        vc.loadOnDevice()
+
+        XCTAssertUIViewControllerDisplayed(isInstance: vc)
+
+        let proceedCalled = expectation(description: "proceedCalled")
+        vc.proceedInWorkflowStorage = { _ in
+            proceedCalled.fulfill()
+        }
+
+        vc.proceedInWorkflow()
+
+        wait(for: [proceedCalled], timeout: TestConstant.timeout)
+    }
+}
+
+extension UIViewController {
+    func loadOnDevice() {
+        // UIUTest's loadForTesting method does not work because it uses the deprecated `keyWindow` property.
+        let window = UIApplication.shared.windows.first
+        window?.removeViewsFromRootViewController()
+
+        window?.rootViewController = self
+        loadViewIfNeeded()
+        view.layoutIfNeeded()
+
+        CATransaction.flush()   // flush pending CoreAnimation operations to display the new view controller
+    }
+}
+
+extension UIKitInteropProgrammaticViewController {
+    var welcomeLabel: UILabel? {
+        view.viewWithAccessibilityIdentifier("welcomeLabel") as? UILabel
+    }
+
+    var saveButton: UIButton? {
+        view.viewWithAccessibilityIdentifier("saveButton") as? UIButton
+    }
+
+    var emailTextField: UITextField? {
+        view.viewWithAccessibilityIdentifier("emailTextField") as? UITextField
+    }
+}
+
+struct FakeContext {
+    let coordinator: (String, String) = ("", "")
```

Comment:
Why this?

Reply:
Because `UIViewControllerRepresentableContent<Thing>` does not expose an initializer.